src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -166,22 +182,24 @@ public function indexAction(Request $request, OrderFilters $filters)
     /**
      * @return array
      */
-    private function getOrderToolbarButtons(): array
-    {
+    private function getOrderToolbarButtons(
+        ShopContext $shopContext
+    ): array {
         $toolbarButtons = [];
 
-        $isSingleShopContext = $this->get('prestashop.adapter.shop.context')->isSingleShopContext();
+        $isSingleShopContext = $shopContext->isSingleShopContext();
```

Comment:
After this PR is merged https://github.com/PrestaShop/PrestaShop/pull/36845 you can use `$this->getShopContext()` instead

Reply:
this is not working as this is not really a shopContext but a regular Context 

` use PrestaShop\PrestaShop\Adapter\Shop\Context as ShopContext; ` 
The `$this->getShopContext()` is missing the method `isSingleShopContext()`